modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/frontend/data/set/view/table/ObjectEntriesTableFDSView.java

```diff
@@ -138,50 +137,67 @@ private void _addFDSTableSchemaField(
 		FDSTableSchemaBuilder fdsTableSchemaBuilder, String fieldName,
 		String label, boolean sortable) {
 
-		FDSTableSchemaField fdsTableSchemaField = null;
-
 		if (Objects.equals(
 				businessType, ObjectFieldConstants.BUSINESS_TYPE_ATTACHMENT) ||
 			Objects.equals(dbType, ObjectFieldConstants.DB_TYPE_CLOB) ||
 			Objects.equals(dbType, ObjectFieldConstants.DB_TYPE_STRING)) {
 
 			StringFDSTableSchemaField stringFDSTableSchemaField =
-				fdsTableSchemaBuilder.addFDSTableSchemaField(
-					StringFDSTableSchemaField.class, fieldName, label);
-
-			stringFDSTableSchemaField.setTruncate(true);
-
-			fdsTableSchemaField = stringFDSTableSchemaField;
+				new StringFDSTableSchemaField();
+
+			fdsTableSchemaBuilder.add(
+				fieldName, label,
+				fdsTableSchemaField -> {
+					if (Validator.isNotNull(contentRenderer)) {
+						stringFDSTableSchemaField.setContentRenderer(
+							contentRenderer);
+					}
+
+					if (_isSortalbe(dbType, sortable)) {
+						stringFDSTableSchemaField.setSortable(true);
+					}
+
+					stringFDSTableSchemaField.setTruncate(true);
+				});
```

Comment:
I don't think this would work. When the lambda argument resolves, it should modify the [newly created `fdsTableSchemaField`](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-impl/src/main/java/com/liferay/frontend/data/set/internal/view/table/FDSTableSchemaBuilderImpl.java#L68-L69). In this PR; we are modifying external `stringFDSTableSchemaField`. I believe we should modify `fdsTableSchemaField`.

Now, if you set `fdsTableSchemaField.setTruncate(true);`, I don't think this will even compile, as this is not in `FDSTableSchemaField` interface. Maybe casting would work? Something like 
```java
StringFDSTableSchemaField stringFDSTableSchemaField = (StringFDSTableSchemaField)fdsTableSchemaField;

stringFDSTableSchemaField.setTruncate(true);
```